src/lib.rs

```diff
@@ -0,0 +1,497 @@
+use std::{collections::HashMap, convert::TryFrom, io::Write, str::FromStr};
+
+use anyhow::{Context, Result};
+use log::info;
+use mbedtls::pk::Pk;
+pub use sdkms::api_model::ObjectType;
+use sdkms::{
+    api_model::{
+        KeyOperations, RsaEncryptionPaddingPolicy, RsaEncryptionPolicy,
+        RsaOptions, RsaSignaturePaddingPolicy, RsaSignaturePolicy, Sobject,
+        SobjectDescriptor, SobjectRequest,
+    },
+    SdkmsClient,
+};
+extern crate sequoia_openpgp as openpgp;
```

Comment:
Since this is an edition 2018 crate, we could write `use sequoia_openpgp as openpgp;` here instead.